src/routes.mli

```diff
@@ -108,12 +113,20 @@ val ( / ) : (('a, 'b) path -> 'c) -> ('d -> ('a, 'b) path) -> 'd -> 'c
 
 (** [l /? r] is used to express the sequence of, parse l followed by parse r and then stop parsing.
     This is used at the end of the route pattern to define how a route should end. The right hand parameter
-    [r] should be a pattern definition that cannot be used in further chains joined by [/] (One such operator is [nil]). *)
-val ( /? ) : (('a, 'b) path -> 'c) -> ('a, 'b) path -> 'c
+    [r] should be a pattern definition that cannot be used in further chains joined by [/].
+
+    [/?] also indicates that a route needs to finish without a trailing slash. *)
+val ( /? ) : ('a -> ('b, 'c) path) -> 'a -> ('b, 'c) target
+
+(** [//?] is similar to [/?] with the difference that it is used to create a route
+    that must finish with a trailing slash.
+
+    @since 0.8.0 *)
+val ( //? ) : ('a -> ('b, 'c) path) -> 'a -> ('b, 'c) target
```

Comment:
should this be: 
```ocaml
val ( //? ) : ('a -> ('b, 'b) path) -> 'a -> ('b, 'b) target
```

to only allow it to be the last call?

Reply:
as discussed on discord, `(//?)` will prevent any further chaining of `paths` so this is safe to do.